theia-go-docker/latest.package.json

```diff
@@ -11,6 +11,7 @@
         }
     },
     "dependencies": {
+        "@theia/cli": "latest",
```

Comment:
I am curious if having @theia/cli as a runtime dependecy vs dev-dependency is required? If so why?

Reply:
I had trouble figuring out why the plugins were not downloaded initially and it's because we build a `--production` version:

https://github.com/theia-ide/theia-apps/blob/9139e67b8c24d949fdee5897d5986975dc15234a/theia-go-docker/Dockerfile#L9

We can either:
- place `@theia/cli` as a **dependency** and keep the production build
- or keep `@theia/cli` as a **devDependency** and remove the production build

Or any other solution I may not have thought about. Any thoughts?

Reply:
Thanks for the explanation. I'm ok to keep as it is.